tabs/applications-setup/tab_data.toml

```
@@ -24,6 +24,10 @@ command = "bash -c \"$(curl -s https://raw.githubusercontent.com/ChrisTitusTech/
 name = "Rofi"
 script = "rofi-setup.sh"
 
+[[data]]
+name = "SDDM Theme"
```

```suggestion
name = "SDDM"
```

One last change. That would be more accurate.
If you know how to, please squash those commits. I can do that if you want https://github.com/ChrisTitusTech/linutil/pull/381#issuecomment-2351081242

---

I agree with @adamperkowski

---

SDDM theme is fine. People can just install sddm without using linutil.

---

Your script doesn't only install the theme. `SDDM Theme` is misleading.

---

> SDDM theme is fine.

I disagree.

---

> The synergy should be kept within the toml file.

And the TUI / UX

---

> > The synergy should be kept within the toml file.
> 
> And the TUI / UX

@mashrukk

---

Okay.